### PR TITLE
[#273] add eclipse-temurin-8-alpine support

### PR DESCRIPTION
**Pull request by yjqg6666**
maven version: 3.8.5
Reference: 
https://github.com/carlossg/docker-maven/tree/master/eclipse-temurin-11
https://hub.docker.com/_/eclipse-temurin?tab=tags&page=1&name=8-jdk-alpine

### CODE REVIEW

**Comment by carlossg** [comment]
Can you also update the README ? thanks
https://github.com/carlossg/docker-maven#adding-new-images

**Comment by yjqg6666** [comment]
@carlossg Updated.

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ See Docker Hub or GitHub Container Registry for an updated list of tags
 * [openjdk-17-slim](https://github.com/carlossg/docker-maven/blob/master/openjdk-17-slim/Dockerfile)
 * [openjdk-18](https://github.com/carlossg/docker-maven/blob/master/openjdk-17/Dockerfile)
 * [openjdk-18-slim](https://github.com/carlossg/docker-maven/blob/master/openjdk-17-slim/Dockerfile)
+* [eclipse-temurin-8-alpine](https://github.com/carlossg/docker-maven/blob/master/eclipse-temurin-8-alpine/Dockerfile)
 * [eclipse-temurin-8](https://github.com/carlossg/docker-maven/blob/master/eclipse-temurin-8/Dockerfile)
```
Comment (carlossg) [inline README.md:25]:
```suggestion
* [eclipse-temurin-8](https://github.com/carlossg/docker-maven/blob/master/eclipse-temurin-8/Dockerfile)
* [eclipse-temurin-8-alpine](https://github.com/carlossg/docker-maven/blob/master/eclipse-temurin-8-alpine/Dockerfile)
```

**Comment by carlossg** [comment]
thanks!

**Pull request merged**